manifests/f/f4exb/sdrangel/7.15.2/f4exb.sdrangel.locale.en-US.yaml

```diff
@@ -0,0 +1,22 @@
+# Created with Komac v1.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: f4exb.sdrangel
+PackageVersion: 7.15.2
+PackageLocale: en-US
+Publisher: f4exb
+PublisherSupportUrl: https://github.com/f4exb/sdrangel/issues
+PackageName: SDRangel
+PackageUrl: https://github.com/f4exb/sdrangel
+License: GPL-3.0
+LicenseUrl: https://github.com/f4exb/sdrangel/blob/master/LICENSE
+ShortDescription: SDR Rx/Tx software for Airspy, Airspy HF+, BladeRF, HackRF, LimeSDR, PlutoSDR, RTL-SDR, SDRplay RSP1 and FunCube
+ReleaseNotes: |-
+  - Allow SigMF plugins to work in Windows.
+    PR #1700 from @srcejon
+  - Only update widgets from UI thread.
+    Fixes #1692.
+    PR #1692 from @srcejon
```

Comment:
```suggestion
ReleaseNotes: |-
  Maintenance release
  - Fixed some spellings. PR https://github.com/f4exb/sdrangel/pull/1742. from @dforsi
  - Rotator Controller: Use floats for az/el offsets. PR https://github.com/f4exb/sdrangel/pull/1744. Fixes https://github.com/f4exb/sdrangel/issues/1739. from @srcejon
  - Switch to codec2-dev repo to access v1.0.3 tag. PR https://github.com/f4exb/sdrangel/pull/1746. Fixes https://github.com/f4exb/sdrangel/issues/1741. from @srcejon
  - Move find_package from external/CMakeLists.txt to root CMakeLists.txt. PR https://github.com/f4exb/sdrangel/pull/1750. from @srcejon
  - Have some libraries and plugins built in the Ubuntu package. PR https://github.com/f4exb/sdrangel/pull/1750. Fixes https://github.com/f4exb/sdrangel/issues/1740. from @srcejon
  - Remove external projects from .deb. PR https://github.com/f4exb/sdrangel/pull/1754. Fixes https://github.com/f4exb/sdrangel/issues/1740. from @srcejon
  - OpenGL code: Clear with alpha=1 to fix https://github.com/f4exb/sdrangel/issues/1735. PR https://github.com/f4exb/sdrangel/issues/1735. from @srcejon
  - DSD demod UI: Connect slot 1/2 on/off buttons. PR https://github.com/f4exb/sdrangel/pull/1758. Fixes https://github.com/f4exb/sdrangel/issues/1752. from @srcejon
  - Use texture() instead of texture2d() in v330 shaders, so they work on Mac. PR https://github.com/f4exb/sdrangel/pull/1759. Fixes https://github.com/f4exb/sdrangel/issues/1757. from @srcejon

  Binary artifacts
  - For more details on artifacts and their contents please read the [quick start](https://github.com/f4exb/sdrangel/wiki/Quick-start) in the Wiki.
  - Note that Ubuntu .deb package is for 22.04. Please note that SSE 4.2 is required.
```